eng/versioning/version_client.txt

```diff
@@ -175,17 +175,17 @@ com.azure:azure-sdk-template;1.1.1234;1.2.2-beta.1
 com.azure:azure-sdk-template-two;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-sdk-template-three;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-spring-data-cosmos;3.39.0;3.40.0-beta.1
-com.azure:azure-storage-blob;12.24.0;12.25.0-beta.2
-com.azure:azure-storage-blob-batch;12.20.0;12.21.0-beta.2
+com.azure:azure-storage-blob;12.24.1;12.25.0-beta.2
+com.azure:azure-storage-blob-batch;12.20.1;12.21.0-beta.2
 com.azure:azure-storage-blob-changefeed;12.0.0-beta.18;12.0.0-beta.19
```

Comment:
Were changefeed and nio not released?

Reply:
They were not.  For some reason, we have not been releasing Change Feed and NIO since May.  This is a topic for our Monday sync, to see if there was a reason we stopped releasing these packages.

Reply:
We don't release them as a part of the automated patches because they are betas still, so any releases are on the Storage team's hands.